resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -748,6 +756,18 @@ export default {
             }
         }
 
+        Events.$on('focusTypeaheadEvent', () => { 
+            this.focusInputFindlabel = false;
+            this.$nextTick(() => {
+                this.focusInputFindlabel = true;
+            });
+           this.focusInputFindlabel = false;
+        });
```

Comment:
```suggestion
```
You don't need to use an event to change `this.focusInputFindLabel`. Simply create a method (e.g., `setFocusFindLabelInput`), move the code there, and call `setFocusFindLabelInput` in `openSidebarLabels()`.